### PR TITLE
Move definitions around. Clarify DKP implementation and blinding mode for test vectors.

### PR DESCRIPTION
**Pull request by chris-wood**
cc @bytemare, @kevinlewi

### CODE REVIEW

**Review by alxdavids — APPROVED** [review]

File: draft-irtf-cfrg-voprf.md [hunk 1]
```diff
@@ -1537,10 +1529,21 @@ Test vectors with batch size B > 1 have inputs separated by a comma
 "Input", "Blind", "BlindedElement", "EvaluationElement", and
 "Output" fields.
 
+Base and verifiable mode use multiplicative and additive blinding as
+described in {{base-client}} and {{verifiable-client}}, respectively.
```
Comment (armfazh) [inline draft-irtf-cfrg-voprf.md:1533]: Although it's verbose, it is easier to understand.

```suggestion
Base mode uses multiplicative blinding while verifiable mode 
uses additive blinding, as described in {{base-client}} and 
{{verifiable-client}}, respectively.
```

File: draft-irtf-cfrg-voprf.md [hunk 2]
```diff
@@ -551,6 +535,14 @@ In this section, we detail the APIs available on the client and server
 - contextString, a domain separation tag taken from the client or server
   context.
 
+The data type `ClientInput` is an opaque byte string of arbitrary length
+no larger than 2^13 octets. `Proof` is a concatenated sequence of two
+`SerializedScalar` values, as shown below.
+
+~~~
+SerializedScalar Proof[2*Ns];
```
Comment (armfazh) [inline draft-irtf-cfrg-voprf.md:543]: Either `opaque Proof[2*Ns]` or `SerializedScalar Proof[2]` is correct.

**Review by armfazh — CHANGES_REQUESTED** [review]
two comments

File: draft-irtf-cfrg-voprf.md (hunk 2 above)
Comment (chris-wood) [reply draft-irtf-cfrg-voprf.md:543]: Sadly, the latter isn’t. The number denotes the byte count, not element count. We were misusing this all along.

Comment (armfazh) [reply draft-irtf-cfrg-voprf.md:543]: It's correct as is. Reference: https://www.rfc-editor.org/rfc/rfc8446.html#section-3.4

**Review by armfazh — APPROVED** [review]

**Pull request merged**